modules/007-registrypackages/images/containerd-altlinux/werf.inc.yaml

```diff
@@ -0,0 +1,44 @@
+{{- $containerd_versions := list }}
+
+{{- range $key, $value := .CandiVersionMap.k8s }}
+  {{- range $key, $versions := $value.bashible.altlinux }}
+    {{- if $versions.containerd.desiredVersion }}
+      {{- $containerd_versions = $versions.containerd.desiredVersion | toString | append $containerd_versions }}
+    {{- end}}
+  {{- end }}
+{{- end }}
+{{- range $value := $containerd_versions | uniq }}
+  {{- $version := $value | toString }}
+  {{- $image_version := $version | replace "containerd=" "" | replace "." "-" | replace "_" "-" }}
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-{{ $image_version }}
+from: {{ $.Images.BASE_SCRATCH }}
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+  add: /
+  to: /
+  includePaths:
+  - containerd_amd64.rpm
+  - install
+  - uninstall
+  before: setup
+docker:
+  LABEL:
+    distro: altlinux
+    version: all
+    containerd: {{ $version | replace "containerd=" "" }}
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+from: {{ $.Images.BASE_ALPINE }}
+git:
+  - add: /modules/007-{{ $.ModuleName }}/images/{{ $.ImageName }}/scripts
+    to: /
+    stageDependencies:
+      setup:
+      - '**/*'
+shell:
+  beforeInstall:
+  - apk add --no-cache curl
+  setup:
+  - curl -sfL http://ftp.altlinux.org/pub/distributions/ALTLinux/p10/branch/files/x86_64/RPMS/containerd-{{ $version | replace "containerd=" "" }}.rpm --output /containerd_amd64.rpm
```

Comment:
I think we need to intall more recent version of containerd due to known issues in old versions. Is it difficult ?